docs/man/ntp.toml.5.md

```diff
@@ -101,6 +101,19 @@ sources.
 :   `pool` mode only. Specifies a list of IP addresses of servers in the pool
     which should not be used. For example: `["127.0.0.1"]`. Empty by default.
 
+`poll-interval-limits` = { `min` = *min*, `max` = *max* } (defaults from `[source-defaults]`)
+:   Specifies the limit on how often a source is queried for a new time. For
+    most instances the defaults will be adequate. The min and max are given as
+    the log2 of the number of seconds (i.e. two to the power of the interval).
+    An interval of 4 equates to 32 seconds, 10 results in an interval of 1024
```

Comment:
2^4 = 16, not 32 :)

This error appears in multiple places within this document and was likely just copy-pasted, but I only noticed it just now